lib/cli-engine/lint-result-cache.js

```diff
@@ -76,17 +115,28 @@ class LintResultCache {
          *    was previously linted
          * If any of these are not true, we will not reuse the lint results.
          */
-
         const fileDescriptor = this.fileEntryCache.getFileDescriptor(filePath);
         const hashOfConfig = hashOfConfigFor(config);
-        const changed = fileDescriptor.changed || fileDescriptor.meta.hashOfConfig !== hashOfConfig;
+        const changed =
+            fileDescriptor.changed ||
+            fileDescriptor.meta.hashOfConfig !== hashOfConfig;
```

Comment:
Will `changed` be `true` if the `cacheStrategy` changes? Related to [this issue](https://github.com/eslint/rfcs/pull/63#issuecomment-676463622) raised in the rfc.